README.md

```diff
@@ -597,8 +597,21 @@ Many thanks for this!
 ### Standard usage
 *	Not specified as required but for working correctly, the epel repository should be available for the 'fping'|'fping6' packages.
 *	Make sure you have sudo installed and configured with: !requiretty.
-*   Make sure that selinux is permissive or disabled.
 
+### SE Linux
```

Comment:
please add a newline after the heading

Reply:
I can recommend https://dlaa.me/markdownlint/ as a linter

Reply:
Actually lint doesn't see that one thing as an issue. And the README is full of it.

Reply:
yep. github changed their parser in the past, so now we try to enforce their rules for new changes.